### DISCUSSION TITLE
Discord?

### DISCUSSION THREAD

**Discussion by Keavon** [webonnx/wonnx#168] [General]
Hi, I am wondering if your project has a Discord (or maybe another chat platform) for communicating?

I run the [Graphite](https://github.com/GraphiteEditor/Graphite) project and we're looking for something to run ML models with WGPU compute shaders so this looks great, but I realize it obviously has some limitations. I'd love to chat more directly, outside the asynchronous conversation flow of a GitHub issue/discussion. Is that something you'd consider creating so users and potential contributors to your project can ask questions and get involved? Graphite's Discord server is used quite effectively for that purpose.

Thanks!

**Comment by pixelspark** [comment]
Hi @Keavon we don’t have a discord and for us the asynchronous communication here has been sufficient up to now. Feel free to drop me an email if you want to discuss something out of band!
